doc/rules/operator/operator_linebreak.rst

```diff
@@ -26,6 +26,15 @@ Allowed values: ``'beginning'``, ``'end'``
 
 Default value: ``'beginning'``
 
+``ignored``
+~~~~~~~~~~~
+
+Which operators to ignore for formatting
```

Comment:
This sentence suggests we will format something and the project is using the term "fixing". Let's change it to "Which operators to ignore" or "List of operators to ignore".